src/views/SystemIntake/index.tsx

```diff
@@ -10,15 +11,24 @@ import { SystemIntakeForm } from 'types/systemIntake';
 import SystemIntakeValidationSchema from 'validations/systemIntakeSchema';
 import flattenErrors from 'utils/flattenErrors';
 import AutoSave from 'components/shared/AutoSave';
-import { initialSystemIntakeForm } from 'data/systemIntake';
-import { v4 as uuidv4 } from 'uuid';
-import { saveSystemIntake } from 'types/routines';
+import { AppState } from 'reducers/rootReducer';
+import {
+  fetchSystemIntake,
+  saveSystemIntake,
+  storeSystemIntakeId
+} from 'types/routines';
 import ContactDetails from './ContactDetails';
 import RequestDetails from './RequestDetails';
 import Review from './Review';
 import './index.scss';
 
-export const SystemIntake = () => {
+export type SystemIDRouterProps = {
```

Comment:
This reminds me we need to nail our naming conventions with the rest of the team. This is fine for now though.